packages/web/src/scss/components/Footer/README.md

```diff
@@ -0,0 +1,239 @@
+
+### Product Logo
+
+Use the [Product Logo][product-logo] component to display the logo of your product.
+
+### Social Media Links
+
+Use the secondary [Button][button] component to create social media links inside a [Flex][flex] container.
+
+```html
+<div class="Flex Flex--row Flex--wrap Flex--alignmentXCenter Flex--alignmentYCenter">
+  <!-- Repeat the `<a>` block for each social media link. -->
+  <a href="https://www.example.com" class="Button Button--secondary Button--medium Button--square">
+    <span class="accessibility-hidden">Facebook</span>
+    <svg width="24" height="24" aria-hidden="true">
+      <use xlink:href="/assets/icons/svg/sprite.svg#logo-facebook" />
+    </svg>
+  </a>
+</div>
+```
+
+### Language Switch
+
+Use the [Select][select] component to create a language switch, or list the languages as country flags in a [Flex][flex]
+layout.
+
+## Secondary Links
+
+This section is optional and consists of a [Flex][flex] layout with secondary links.
+
+```html
+<ul
+  class="Flex Flex--column Flex--tablet--row Flex--wrap Flex--alignmentXStretch Flex--tablet--alignmentXCenter Flex--alignmentYStretch"
+  style="--flex-spacing: var(--spirit-space-500); --flex-spacing-tablet: var(--spirit-space-700)"
+>
+  <!-- Repeat the `<li>` block for each secondary link. -->
+  <li>
+    <a href="https://www.example.com" class="link-secondary">Legal notice</a>
+  </li>
+</ul>
+```
+
+## Full Example
+
+```html
+<footer class="bg-cover pt-1100 pb-1000">
+  <div class="Container">
+    <!-- Grid with navigation links -->
+    <div
+      class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-4"
+      style="--grid-spacing-x: var(--spirit-space-800); --grid-spacing-y: var(--spirit-space-800)"
+    >
+      <!-- Repeat the `<nav>` block as many times as needed. -->
+      <nav aria-labelledby="footer-part-one">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-part-one">Section headline</h3>
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="https://www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="https://www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="https://www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- Divider -->
+    <hr class="Divider my-900" />
+
+    <!-- Grid with product logo, social media links and language switch -->
+    <div
+      class="Grid Grid--cols-1 Grid--desktop--cols-3 Grid--alignmentXCenter Grid--desktop--alignmentXStretch Grid--alignmentYCenter"
+      style="--grid-spacing-x: var(--spirit-space-900); --grid-spacing-y: var(--spirit-space-900)"
+    >
+      <!-- Product logo -->
+      <div class="text-desktop-left">
+        <a href="https://www.example.com">
+          <div aria-label="Logo of the JobBoard" class="UNSTABLE_ProductLogo">
```

Comment:
```suggestion
          <div aria-label="Logo of the Product" class="UNSTABLE_ProductLogo">
```

I know our main focus are job boards, but maybe this should be generalized?

Reply:
I think not in this case 🙂 🤷🏻‍♂️ 

![image](https://github.com/user-attachments/assets/28fbe200-9122-40b5-b323-d323c514fcdd)